### ISSUE TITLE
Download without registration

### ISSUE THREAD

**Issue by habi** [horosproject/horos#603]
The [download button](https://horosproject.org/download-horos/) in the top of your page forces me to submit a working email address to download Horos, which I don't think is necessary.
I'd like to be able to download the app without registration, e.g. by going to https://horosproject.org/download

**Comment by TheDagger** [comment]
I agree, it's a bit strange for a program that is free? People will just add spam/fake emails to this anyways.  If this were a program to make money that is understandable, but I think that is OsiriX's job? Maybe they use the emails to make extra money for non-profit reasons? Maybe it's to force the downloader to see important update information? I'm not sure since I'm not involved in any form, just giving my 2 cents.

**Comment by spalte** [comment]
Hello,

I’m not involved with Puview/Horos, but I have written a lot of code in the strange space that is medically-relevant open-source and research software. You want to share work that can be helpful to others, but can not sustain the costs associated with being a registered medical device. It is a rather uncomfortable place from a legal point of view. When working in this murky space, given that there is no good legal guidance, I rely even more intensely on my sense of morals and moral obligations. When I write software that could potentially be used by others to make medical decisions with important consequences, I try real hard to not have bugs that might affect a patient down the line. I also try to get a good grasp on who my users are, so that in the catastrophic situation where there might be a bug that could put people in danger, I at least can make an effort to limit the damages. This includes direct users, as well as commercial vendors that might have integrated my work into their registered medial devices.

Again, I am not involved with Purview or Horos, and I don’t have any particular insight into why they decided to collect email addresses. But I don't mind giving them my email address, because it is what I would do if I were in their shoes.

Cheers,
Joël

**Comment by diimdeep** [comment]
Let people to download and try your software without this ridiculous 'infowall' page, if software is good they will contribute and leave personal information voluntarily. In fact, you should measure how many people bounced off without downloading anything, unable to bypass this page.

**Comment by pszypowicz** [comment]
Today I tried to register and definitely did not receive download link via email (it is not in spam also). Please let users just download your app directly from your website.
